src/ngsi-converter.js

```diff
@@ -0,0 +1,136 @@
+const URI = require("uri-js");
```

Comment:
better to name this `normalized2LD.js` to resemble the existing script in Python